Lib/json/tool.py

```diff
@@ -15,6 +15,25 @@
 import json
 import sys
 
+def parse_json_from_filehandle(infile, sort_keys):
```

Comment:
These functions are a bit repetitive and add a lot of unnecessary lines of code. I think they could be avoided by keeping the functionality in the original location. The code could become more like:

```
sort_keys = options.sort_keys
jsonlines = options.jsonlines
with infile:
    if jsonlines:
        if sort_keys:
            objs = [json.loads(line) for line in infile]
        else:
           ...
    else:
        if sort_keys:
            objs = [json.load(infile)]
        else:
            ...
```
This keeps the code more concise.